### DISCUSSION TITLE
Documentation of the Android app - "Arrow up - bar"

### DISCUSSION THREAD

**Discussion by M-Stenzel** [owntracks/android#1822] [Q&A]
Hi Community,

really happy to have found this cool software!

But, where to find documentation regarding the main window?

This is what I have:

![image](https://github.com/user-attachments/assets/b67f16ce-a80f-49cb-92e7-4cf9199807ea)

What is the function of the "Arrow up - bar" icon in the right upper border? Pressing this does not change anything...

Martin.

**Comment by jpmens** [comment]
That arrow publishes your current location to the endpoint you've configured. The location payload contains a `"t": "m"` indicating a Manual publish.

**Reply by M-Stenzel** [reply]
Oh, that was quick, thank you. O. k. know I know...
